app/app/vhosts/new/template.hbs

```diff
@@ -50,15 +45,34 @@
             </div>
           </div>
 
-          <div class="form-group">
-            <label>Certificate</label>
-            {{textarea class="form-control" value=model.certificate name="certificate" rows=10}}
-          </div>
+          {{#if showAddNewCertificate}}
```

Comment:
This section seems to be exactly the same between the `new` and `edit` templates, can we tweak this to be a shared component instead?